src/components/gb-card-1.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../style/gb-style.css';
+const CardOneLi = ({ follower, category, background }) => {
```

Comment:
Make it's own component, it should not leave in here, in the same file with the parent component

Reply:
Done!